tools/medenv/med_environ.xml

```diff
@@ -0,0 +1,129 @@
+    </outputs>
+
+	<tests>
+		<test expect_num_outputs="1">
+			<param name="input" value="reef_life_subset.csv"/>
+			<param name="env_variables" value="bathymetry"/> <!-- This test is requesting etopo only -->
+			<param name="lat_key" value="SiteLat"/>
+			<param name="long_key" value="SiteLong"/>
+			<param name="tol_spatial_coordinates" value="0.2"/>
+			<param name="depth_key" value="Depth"/>
+			<param name="date_key" value="SurveyDate"/>
+			<param name="cmems_username" value="xxxx"/> <!-- Not used for this test -->
+			<param name="cmems_password" value="xxxx"/> <!-- Not used for this test -->
+			<param name="verbose" value="False"/>
+			<output name="out_file">
+				<assert_contents>
+					<has_n_lines n="100"/>
+				</assert_contents>
+			</output>
+		</test>
+    </tests>
+
+	<help>
+
+		**What it does**
+
+		This script allows to augment your dataset by adding some environmental variables grabbed from CMEMS, etopo, woa. It is a galaxy tool built on the medenv python package available at https://github.com/jeremyfix/medenv .
+
+		We expect the dataset you provide to contain the coordinates of the observation : latitude, longitude, depth, time. The time is expected in ISO 8601 : YYYY-MM-DDTHH:MM:SSZ
+
+        The `cmems_username` and `cmems_password` is required only if you want to add variables provided by CMEMS.
+
+        The CMEMS data are collected from the following products :
+
+			- med-cmcc : https://resources.marine.copernicus.eu/product-detail/MEDSEA_MULTIYEAR_PHY_006_004/INFORMATION
+			- med-ogs : https://resources.marine.copernicus.eu/product-detail/MEDSEA_MULTIYEAR_BGC_006_008/INFORMATION
+
+		The bathymetry is obtained with the ETOPO at : 
+
+			- https://www.ngdc.noaa.gov/mgg/global/relief/ETOPO1/data/bedrock/grid_registered/netcdf/ETOPO1_Bed_g_gmt4.grd.gz
+
+        We request data around the point with a given tolerance and average the collected values. The medenv package does not force that averaging and could return a raster. A latter update of the galaxy tool could take this into account.
+
+		The features you can request are the following, some indexed by depth :
+
+            - "bathymetry", (etopo)
+            - "temperature",
+            - "salinity",
+            - "chlorophyl-a",
+            - "nitrate",
+            - "phosphate",
+            - "ammonium",
+            - "phytoplankton-carbon-biomass",
+            - "oxygen",
+            - "net-primary-production",
+            - "ph",
+            - "alkalinity",
+            - "dissolved-inorganic-carbon",
+            - "northward-water-velocity",
+            - "eastward-water-velocity".
+
+		and others not indexed by depth : 
+
+            - "mixed-layer-thickness",
+            - "sea-surface-temperature",
+            - "sea-surface-salinity",
+            - "sea-surface-above-geoid",
+            - "surface-partial-pressure-co2",
+            - "surface-co2-flux".
+
+		</help>
+    <citations>
```

Comment:
you can use type=bibtex here and cite your github repo if you like

Reply:
Done as well ; Although not cited yet assigned a DOI :)